mindmeld/text_preparation/text_preparation_pipeline.py

```diff
@@ -335,6 +346,107 @@ def find_mindmeld_annotation_re_matches(text):
         """
         return list(MINDMELD_ANNOTATION_PATTERN.finditer(text))
 
+    @staticmethod
+    def calc_unannotated_spans(text):
+        """Calculates the spans of text that exclude mindmeld entity annotations.
```

Comment:
Just a nice to have: 
Going by the naming, I assumed this function (and other variables used named similarly) will return all spans other than entity spans. 
Can't think of a better alternative though - `calc_spans_without_annotation_markers()` maybe?

Reply:
Clarified the definition/explanation as discussed